app/grandchallenge/participants/models.py

```diff
@@ -4,7 +4,7 @@
 from grandchallenge.challenges.models import Challenge
 
 
-class RegistrationRequest(models.Model):
+class RequestBase(models.Model):
```

Comment:
Could this be moved to the `core` application to avoid any cross-app dependencies?